### PR TITLE
Sign in ui is done

### PR DESCRIPTION
**Pull request by Joldubaev**
## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: app/lib/modules/login/view/login_view.dart [hunk 1]
```diff
@@ -24,6 +24,9 @@ class LoginView extends StatelessWidget {
           ),
         ),
       ),
+      floatingActionButton: FloatingActionButton(
+        onPressed: () => Navigator.pushNamed(context, AppRouter.signIn),
+      ),
```
Comment (Eldar2021) [inline app/lib/modules/login/view/login_view.dart:29]: Suranych munu komentariyge alyp, ustunu sign view ga baryy uçhun munu active kyl gegendi tushundurgon comment jazynyz :) (developerler tushunsun)

File: app/lib/l10n/arb/app_tr.arb [hunk 2]
```diff
@@ -56,5 +56,11 @@
...
     "profileDevsDesc": "Uygulamamızda herhangi bir teknik kusur fark ettiyseniz veya uygulamaya yeni özellikler eklemek için ilginç fikirleriniz varsa, sizden haber almaktan her zaman mutluluk duyarız! GitHub'da bizimle iletişime geçin. Lütfen utanma! Bu uygulamayı birlikte geliştirmekle çok ilgileniyoruz.",
     "allDonePages": "Uygulamamızdan okunan toplam Kuran sayfa sayısı",
-    "allDonePagesDesc": "Allah şu Kur’an’la bazı kavimleri yükseltir, bazılarını da alçaltır."
+    "allDonePagesDesc": "Allah şu Kur’an’la bazı kavimleri yükseltir, bazılarını da alçaltır.",
+    "welcome": "Hoş geldiniz",
+    "google": "Google",
+    "orsigninwith": "---Veya ile giriş yap---",
+    "apple": "Apple",
+    "signinwithoutanaccount": "Hesapsız giriş yap",
+    "privacypolicy": "Gizlilik Politikası"
```
Comment (Eldar2021) [inline app/lib/l10n/arb/app_tr.arb:65]: Bul jerde camelCase koldonuunu sunushtaim
M:
1)  orsigninwith->orSigninWith
....

File: app/lib/l10n/arb/app_en.arb [hunk 3]
```diff
@@ -56,5 +56,11 @@
...
     "profileDevsDesc": "If you've noticed any technical flaws in our app, or if you have interesting ideas for adding new features to the app, we're always happy to hear from you! Get in touch with us on GitHub. Please don't be shy! We are very interested in developing this app together.",
     "allDonePages": "Total number of Quran pages read from our app",
-    "allDonePagesDesc": "Allah şu Kur’an’la bazı kavimleri yükseltir, bazılarını da alçaltır."
+    "allDonePagesDesc": "Allah şu Kur’an’la bazı kavimleri yükseltir, bazılarını da alçaltır.",
+    "welcome": "Welcome",
+    "google": "Google",
+    "orsigninwith": "---Or sign in with---",
+    "apple": "Apple",
+    "signinwithoutanaccount": "Sign in without an account",
+    "privacypolicy": "Privacy Policy"
```
Comment (Eldar2021) [inline app/lib/l10n/arb/app_en.arb:65]: Use camelCase please :)

File: app/lib/modules/login/view/sign_in_view.dart [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+          CustomButtonWithIcon(
+            icon: const Icon(FontAwesomeIcons.apple),
+            onPressed: () {},
+            text: context.l10n.apple,
+          ),
+          const SizedBox(height: 90),
+          CustomButton(onPressed: () {}, text: context.l10n.signinwithoutanaccount),
+          const Spacer(),
+          TextButton(
+            onPressed: () {},
```
Comment (Eldar2021) [inline app/lib/modules/login/view/sign_in_view.dart:51]: Eger oorchuluk kelbese bir privicy policy dokumen tuzup any bol jerge link katary ishtete alasyzby (Ayabai kubanmakbyz, aldyn ala rahnat 👍 )

File: app/pubspec.yaml [hunk 5]
```diff
@@ -31,6 +31,7 @@ dependencies:
   google_fonts: ^5.0.0
   mq_storage:
     path: ../packages/mq_storage/
+  font_awesome_flutter: ^10.4.0
```
Comment (Eldar2021) [inline app/pubspec.yaml:34]: Suranych bul jerden ochurup 32 line ga koyo alasyzby :)

**Review by Eldar2021 — COMMENTED** [review]
Sonun chong rahmat sizge! Kee bir gana sunushtarym bar :)

File: app/lib/modules/login/view/sign_in_view.dart (hunk 4 above)
Comment (Joldubaev) [reply app/lib/modules/login/view/sign_in_view.dart:51]: I made but you should  change link

File: app/lib/constants/api/api_const.dart [hunk 6]
```diff
@@ -21,4 +21,5 @@ class ApiConst {
   static Map<String, String> authMap(String token) => {'Authorization': 'Bearer $token'};
 
   static const urlGitHub = 'https://github.com/Eldar2021/my_quran';
+  static const provicyPolicy = 'https://github.com/Eldar2021/my_quran';
```
Comment (Eldar2021) [inline app/lib/constants/api/api_const.dart:24]: Suranych ushul jerge todo jaza alasyzby. Kiyin linkti almashtyr degen siyaktuu. [dart comments](https://dart.dev/language/comments)

File: app/lib/l10n/arb/app_en.arb [hunk 7]
```diff
@@ -56,5 +56,11 @@
     "profileDevs": "For developers",
     "profileDevsDesc": "If you've noticed any technical flaws in our app, or if you have interesting ideas for adding new features to the app, we're always happy to hear from you! Get in touch with us on GitHub. Please don't be shy! We are very interested in developing this app together.",
     "allDonePages": "Total number of Quran pages read from our app",
-    "allDonePagesDesc": "Allah şu Kur’an’la bazı kavimleri yükseltir, bazılarını da alçaltır."
+    "allDonePagesDesc": "Allah şu Kur’an’la bazı kavimleri yükseltir, bazılarını da alçaltır.",
+    "welcome": "Welcome",
+    "google": "Google",
+    "orSignInWith": "---Or sign in with---",
```
Comment (Eldar2021) [inline app/lib/l10n/arb/app_en.arb:62]: 👍

File: app/pubspec.yaml [hunk 8]
```diff
@@ -29,8 +29,10 @@ dependencies:
   url_launcher: ^6.1.11
   json_annotation: ^4.8.1
   google_fonts: ^5.0.0
+  font_awesome_flutter: ^10.4.0
   mq_storage:
     path: ../packages/mq_storage/
+
```
Comment (Eldar2021) [inline app/pubspec.yaml:35]: Please delete empty line :)

File: app/lib/modules/login/view/login_view.dart [hunk 9]
```diff
@@ -24,6 +24,10 @@ class LoginView extends StatelessWidget {
           ),
         ),
       ),
+
+      /// This code assumes that the AppRouter class is defined
+      /// elsewhere and contains a static member named signIn
+      /// that represents the route to navigate to.git
```
Comment (Eldar2021) [inline app/lib/modules/login/view/login_view.dart:30]: Tuura ems tushundurup alganym uchun ayabai kechirim suraym. :(
```suggestion
      // SignIn sayfasını görmek için bunu active edebilirsiniz (tabi ingilizce olmalı)
      // floatingActionButton: FloatingActionButton(
      //   onPressed: () => Navigator.pushNamed(context, AppRouter.signIn),
      // ),
```

File: app/lib/modules/login/view/sign_in_view.dart [hunk 10]
```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+import 'package:my_quran/components/components.dart';
+import 'package:my_quran/constants/contants.dart';
+import 'package:my_quran/core/core.dart';
+import 'package:my_quran/l10n/l10.dart';
+import 'package:my_quran/theme/theme.dart';
+
+class SignInView extends StatefulWidget {
```
Comment (Eldar2021) [inline app/lib/modules/login/view/sign_in_view.dart:10]: Please use  `StatelessWidget` :)

**Review by Eldar2021 — COMMENTED** [review]
Sabyrynyz uchun rahmat :) Sonun UI kelip jatkanyna ayabai kubanychtamyn. Az kaldy 👏

File: app/analysis_options.yaml [hunk 11]
```diff
@@ -12,3 +12,7 @@ analyzer:
     - "**/*.g.dart"
     - "**/*.freezed.dart"
     - "bin/cache/**"
+
+  errors:
+   todo: false
+   fixme: false
```
Comment (Eldar2021) [inline app/analysis_options.yaml:18]: Cool! 🤙

File: app/lib/constants/api/api_const.dart [hunk 12]
```diff
@@ -21,4 +21,7 @@ class ApiConst {
   static Map<String, String> authMap(String token) => {'Authorization': 'Bearer $token'};
 
   static const urlGitHub = 'https://github.com/Eldar2021/my_quran';
+
+  // TODO(eldiiar): Replace the privacy policy URL with the actual [URL] of your privacy policy page.
```
Comment (Eldar2021) [inline app/lib/constants/api/api_const.dart:25]: Nice TODO 👍

File: app/lib/modules/login/view/login_view.dart [hunk 13]
```diff
@@ -24,6 +24,10 @@ class LoginView extends StatelessWidget {
           ),
         ),
       ),
+      // You can enable it to see the SignIn page
+      // floatingActionButton: FloatingActionButton(
+      //   onPressed: () => Navigator.pushNamed(context, AppRouter.signIn),
+      // ),
```
Comment (Eldar2021) [inline app/lib/modules/login/view/login_view.dart:30]: Tam istediğim gibi teşekkürler :)

File: app/lib/modules/login/view/sign_in_view.dart
Comment (Eldar2021) [inline app/lib/modules/login/view/sign_in_view.dart:1]: Bu dosyayı henüz test etmedim, ama temiz ve iyi görünüyor. O yüzden onaylıyorum. Elinize sağlık abi.

**Review by Eldar2021 — APPROVED** [review]
Finally we are ready to merge this thanks a lot.

**Pull request merged**